lms/templates/components/header/header.underscore

```diff
@@ -2,15 +2,15 @@
     <div class="page-header-main">
         <div class="sr-is-focusable" tabindex="-1"></div>
         <% if (breadcrumbs !== null && breadcrumbs.length > 0) { %>
-            <nav class="breadcrumbs" aria-label="<%- nav_aria_label %>">
+            <nav class="breadcrumbs" aria-label="<%= HtmlUtils.ensureHtml(nav_aria_label) %>">
```

Comment:
Why did this change?  I don't think this is HTML, so why not use the `<%-`?  Also, if we moved the gettext() call here, it would be even for clear.